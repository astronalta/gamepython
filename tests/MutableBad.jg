
main() Int {
    y Int = 0
    y = 20
    x = 40 
    x = 0
    x = 10 
    x = 30 
    ret 0
}
