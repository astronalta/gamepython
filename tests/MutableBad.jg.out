MutableBad.jg:4:5: Value assigned to immutable variable 'y'
MutableBad.jg:6:5: Value assigned to immutable variable 'x'
MutableBad.jg:7:5: Value assigned to immutable variable 'x'
MutableBad.jg:8:5: Value assigned to immutable variable 'x'
